### ISSUE TITLE
request: blur background

### ISSUE THREAD

**Issue by GeneralMidi** [dumbeau/AutoHotPie#13]
request

can you perhaps blur the background to help the menu stand out?

![image](https://user-images.githubusercontent.com/47597511/144416967-23ce06a6-4bfd-40f9-ae68-863ec82bc63e.png)

**Comment by dumbeau** [comment]
Yeah I see the need for this, either a subtle, heavily blurred drop shadow/highlight or blurring the background could work here.  I see this being a global menu setting like the font selection.

**Comment by mmikeww** [comment]
the skin that i want to build is the wacom radial menu style:
https://www.ctrlpaint.com/blog/the-radial-menu

**Comment by GeneralMidi** [comment]
Oh very nice. I'm surprised this is being looked at, I thought the idea was dead. I'll be following closely!
